tools/mmp/driver.cs

```diff
@@ -408,9 +408,20 @@ static void Main2 (string [] args)
 			if (!targetFramework.HasValue)
 				targetFramework = TargetFramework.Default;
 
-			if (TargetFramework.Identifier == TargetFramework.Xamarin_Mac_2_0.Identifier) {
+			if (TargetFramework == TargetFramework.Xamarin_Mac_2_0_Mobile || TargetFramework.Version == TargetFramework.Xamarin_Mac_2_0_Mobile.Version) {
```

Comment:
You'd think. Building tests I ran into one place where we manually were creating a TargetFramework of:

Xamarin.Mac,v2.0,(null)

and TargetFramework.Identifier == TargetFramework.Xamarin_Mac_2_0.Identifier would fail, since (null) does not equal Mobile.

So I baked in the assumption that if your v2.0 then you really meant mobile.

Should I comment in code?